knip.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+    "workspaces": {
+        ".": {
+            // These entries are complementary to the ones found in package.json
+            "entry": [
+                "lib/rules/index.js",
+                "tools/internal-rules/index.js",
+                "tools/update-rule-types.js",
+                "wdio.conf.js" // Remove when Knip has a wdio plugin
```

Comment:
[Request] Could you link a tracking issue in the comment? Unless another team member hates this _(I'm new here!)_, I prefer having them there to make it easy to discover.

Reply:
In case the team likes this idea (Knip in general, that plugin specifically, etc), feel free to open an RFC at the Knip repo.

Reply:
```suggestion
                // https://github.com/webpro/knip/issues/464
                // Remove when Knip has a wdio plugin
                "wdio.conf.js" 
```